reads2transcriptome/ci/samplesheet.csv

```diff
@@ -0,0 +1,7 @@
+"sample","fastq_1","fastq_2","strandedness","run_accession","experiment_accession","sample_accession","secondary_sample_accession","study_accession","secondary_study_accession","submission_accession","run_alias","experiment_alias","sample_alias","study_alias","library_layout","library_selection","library_source","library_strategy","library_name","instrument_model","instrument_platform","base_count","read_count","tax_id","scientific_name","sample_title","experiment_title","study_title","sample_description","fastq_md5","fastq_bytes","fastq_ftp","fastq_galaxy","fastq_aspera","assembly_group"
+"SRX15140807","https://raw.githubusercontent.com/Arcadia-Science/test-datasets/gm/reads2transcriptome-ci/reads2transcriptome/ci/SRX15140807_SRR19070014.fastq.gz","","auto","SRR19070014","SRX15140807","SAMN28065452","SRS12884154","PRJNA834823","SRP373454","SRA1414352","subreads_ccs.flnc.polished.hq.hq.fastq","Aa_af_sg_isoseq_1","Aa_adultfemale_salivarygland_isoseqtranscriptome_1","PRJNA834823","SINGLE","Oligo-dT","TRANSCRIPTOMIC","RNA-Seq","Aa_af_sg_isoseq_1","Sequel II","PACBIO_SMRT","1343427230","307541","6943","Amblyomma americanum","Invertebrate sample from Amblyomma americanum","Sequel II sequencing: Long-read transcriptome of female Amblyomma americanum salivary glands","Amblyomma americanum salivary gland long-read transcriptome","Invertebrate sample from Amblyomma americanum","223a5e9f9ff5836ca3c2f5c4974ce575","464748884","ftp.sra.ebi.ac.uk/vol1/fastq/SRR190/014/SRR19070014/SRR19070014_subreads.fastq.gz","ftp.sra.ebi.ac.uk/vol1/fastq/SRR190/014/SRR19070014/SRR19070014_subreads.fastq.gz","fasp.sra.ebi.ac.uk:/vol1/fastq/SRR190/014/SRR19070014/SRR19070014_subreads.fastq.gz","isoseq"
```

Comment:
I'd convert these to read from `main` instead of your branch. As I am sure your branch will eventually be deleted.

Reply:
yes, this was done temporarily in ae5479f so that I can test whether the CI is working before merging the PR (review required). It will be reverted to 29f1a14 the PR is approved.